user_notification/models.py

```diff
@@ -24,22 +26,67 @@ class JobAlert(models.Model):
     def __str__(self):
         return str(self.user_account_id)
 
-    # Calculate the date of the next alert
     @classmethod
     def calc_check_date(cls, last_check_date, frequency_in_days):
+        # Calculate the date of the next alert
         if frequency_in_days < 1:
             myError = ValueError(
                 'frequency_in_days should be Greater than or equal to 1')
             raise myError
```

Comment:
No need for "myError" variable, just do `raise ValueError('frequency_in_days should be Greater than or equal to 1')`

Reply:
fixed